src/pages/ni/config/RhConfig.vue

```diff
@@ -185,22 +209,89 @@
+        :options="urssafCodes" :error="$v.newEstablishment.urssafCode.$error" :error-label="establishmentUrssafCodeError"
+        @blur="$v.newEstablishment.urssafCode.$touch" required-field />
+      <template slot="footer">
+        <q-btn no-caps class="full-width modal-btn" label="Ajouter un établissement" icon-right="add" color="primary"
+          :disable="!$v.newEstablishment.$anyDirty || $v.newEstablishment.$invalid" :loading="loading" @click="createNewEstablishment" />
+      </template>
+    </ni-modal>
+
+    <!-- Establishment edition modal -->
+    <ni-modal v-model="establishmentEditionModal" @hide="resetEstablishmentEditionModal">
+      <template slot="title">
+        Ajouter un <span class="text-weight-bold">établissement</span>
+      </template>
+      <ni-input in-modal caption="Nom" v-model="editedEstablishment.name" :error="$v.editedEstablishment.name.$error"
+        :error-label="establishmentNameError" @blur="$v.editedEstablishment.name.$touch" required-field />
+      <ni-input in-modal caption="SIRET" v-model="editedEstablishment.siret" :error="$v.editedEstablishment.siret.$error"
+        :error-label="establishmentSiretError" @blur="$v.editedEstablishment.siret.$touch" required-field />
+      <ni-search-address in-modal v-model="editedEstablishment.address" color="white" @blur="$v.editedEstablishment.address.$touch"
+        :error-label="establishmentAddressError" :error="$v.editedEstablishment.address.$error" required-field />
+      <ni-input in-modal caption="Téléphone" v-model="editedEstablishment.phone" :error="$v.editedEstablishment.phone.$error"
+        :error-label="establishmentPhoneError" @blur="$v.editedEstablishment.phone.$touch" required-field />
+      <ni-select in-modal caption="Service de santé du travail" v-model="editedEstablishment.workHealthService"
+        :options="workHealthServices" :error="$v.editedEstablishment.workHealthService.$error" :error-label="establishmentWhsError"
+        @blur="$v.editedEstablishment.workHealthService.$touch" required-field />
+      <ni-select in-modal caption="Code URSSAF" v-model="editedEstablishment.urssafCode"
+        :options="urssafCodes" :error="$v.editedEstablishment.urssafCode.$error" :error-label="establishmentUrssafCodeError"
+        @blur="$v.editedEstablishment.urssafCode.$touch" required-field />
+      <template slot="footer">
+        <q-btn no-caps class="full-width modal-btn" label="Editer l'établissement" icon-right="add" color="primary"
+          :disable="$v.editedEstablishment.$invalid" :loading="loading" @click="updateEstablishment" />
+      </template>
+    </ni-modal>
   </q-page>
 </template>
 
 <script>
-import { required, maxValue } from 'vuelidate/lib/validators';
+import { required, maxValue, maxLength } from 'vuelidate/lib/validators';
+import get from 'lodash/get';
+import cloneDeep from 'lodash/cloneDeep';
+import pick from 'lodash/pick';
```

Comment:
Normalement tu n'as pas besoin d'importer lodash dans le front, il est accessible par `this.$_`

Reply:
En fait, je voulais le faire pour des raisons de perfs mais c'est vrai que puisque le paquet est déjà chargé autant l'utiliser ^^